cedar-policy/src/frontend/utils.rs

```diff
@@ -20,11 +20,13 @@ use std::collections::HashMap;
 
 #[derive(Debug, Serialize, Deserialize)]
 #[serde(untagged)]
+#[serde(expecting = "policy map must have no duplicate IDs,")]
```

Comment:
I expect this will give a bad error message if the policy specification happens to be an int or null or something else. Maybe best to spell it out "policies as a concatenated string or multiple policies as a hashmap where the policy Id is the key with no duplicate IDs". Although maybe that's a bit wordy.